### PR TITLE
Use correct DB connection for postgres

### PR DESCRIPTION
**Pull request by jhoward1994**

### CODE REVIEW

File: packages/core/admin/ee/server/services/review-workflows/review-workflows.js [hunk 1]
```diff
@@ -108,23 +108,22 @@ function enableReviewWorkflow({ strapi }) {
...
           [joinTable.joinColumn.name]: firstStage.id,
-          [typeColumn.name]: connection.raw('?', [contentTypeUID]),
+          [typeColumn.name]: strapi.db.connection.raw('?', [contentTypeUID]),
         })
         .leftJoin(`${joinTable.name} AS jointable`, function () {
           this.on('entity.id', '=', `jointable.${idColumn.name}`).andOn(
             `jointable.${typeColumn.name}`,
             '=',
-            connection.raw('?', [contentTypeUID])
+            strapi.db.connection.raw('?', [contentTypeUID])
```
Comment (Marc-Roig) [inline packages/core/admin/ee/server/services/review-workflows/review-workflows.js:126]: would it be a problem if we did

```js
.select({
          [idColumn.name]: 'entity.id',
          field: ENTITY_STAGE_ATTRIBUTE,
          order: 1,
          [joinTable.joinColumn.name]: firstStage.id,
          [typeColumn.name]: contentTypeUID,
        })
```

File: packages/core/admin/ee/server/services/review-workflows/review-workflows.js [hunk 2]
```diff
@@ -108,23 +108,22 @@ function enableReviewWorkflow({ strapi }) {
...
 
-      const connection = strapi.db.getConnection();
-
       // Execute an SQL query to insert records into the join table mapping the specified content type with the first stage of the default workflow.
       // Only entities that do not have a record in the join table yet are selected.
-      const selectStatement = connection
+      const selectStatement = strapi.db
+        .getConnection()
         .select({
           [idColumn.name]: 'entity.id',
```
Comment (Marc-Roig) [inline packages/core/admin/ee/server/services/review-workflows/review-workflows.js:116]: is this ok? shouldn't be 
```suggestion
          [idColumn.name]: entity.id,
```

Comment (jhoward1994) [reply packages/core/admin/ee/server/services/review-workflows/review-workflows.js:116]: I think it should be in quotes. entity is an SQL alias and not a JS variable

File: packages/core/admin/ee/server/services/review-workflows/review-workflows.js (hunk 1 above)
Comment (jhoward1994) [reply packages/core/admin/ee/server/services/review-workflows/review-workflows.js:126]: I can't seem to get it working without using query bindings

```
insert into `strapi_workflows_stages_related_morphs` (related_id,field,`order`,workflow_stage_id,related_type)  select `entity`.`id` as `related_id`, `strapi_reviewWorkflows_stage` as `field`, 1 as `order`, 1 as `workflow_stage_id`, `api::product`.`product` as `related_type` from `products` as `entity` left join `strapi_workflows_stages_related_morphs` as `jointable` on `entity`.`id` = `jointable`.`related_id` and `jointable`.`related_type` = `api::product`.`product` where `jointable`.`related_id` is null - no such column: strapi_reviewWorkflows_stage
```

File: packages/core/admin/ee/server/services/review-workflows/review-workflows.js (hunk 2 above)
Comment (Marc-Roig) [reply packages/core/admin/ee/server/services/review-workflows/review-workflows.js:116]: oh my bad! completely missed it was an alias just below

File: packages/core/admin/ee/server/services/review-workflows/review-workflows.js (hunk 1 above)
Comment (Marc-Roig) [reply packages/core/admin/ee/server/services/review-workflows/review-workflows.js:126]: 👌 interesting

**Review by Marc-Roig — APPROVED** [review]

**Pull request merged**